source/common/http/conn_manager_config.h

```diff
@@ -64,6 +64,7 @@ namespace Http {
   COUNTER  (downstream_rq_overload_close)                                                          \
   COUNTER  (downstream_rq_timeout)                                                            \
   COUNTER  (rs_too_large)
+// Do I need a stat addition for this new feature?
```

Comment:
ptal at this reviewer

Reply:
I think you can skip a stat for now since we didn't have one before. If you want to add a stat feel free though.